### PR TITLE
Feature/1 implement database

### PR DESCRIPTION
**Pull request by Liosh-a**
The database was successfully created on a local computer.

### CODE REVIEW

File: BobManager.API/BobManager.DataAccess/ApplicationContext.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+﻿using BobManager.DataAccess.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace BobManager.DataAccess
+{
+    public class ApplicationContext : DbContext
+    {
+        public ApplicationContext(DbContextOptions<ApplicationContext> options) : base(options)
+        {
+
```
Comment (ander21) [inline BobManager.API/BobManager.DataAccess/ApplicationContext.cs]: Remove empty space please.

File: BobManager.API/BobManager.DataAccess/ApplicationContext.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                .HasOne(sc => sc.User)
+                .WithMany(c => c.Groups)
+                .HasForeignKey(sc => sc.UserId);
+
+            modelBuilder.Entity<UsersGroup>()
+                .HasOne(sc => sc.GroupRole)
+                .WithMany(c => c.UsersGroups)
+                .HasForeignKey(sc => sc.GroupRoleId);
+        }
+
```
Comment (ander21) [inline BobManager.API/BobManager.DataAccess/ApplicationContext.cs]: Remove empty space, please!

**Review by ander21 — COMMENTED** [review]
Resolve comments, please!

File: BobManager.API/BobManager.API/Startup.cs [hunk 3]
```diff
@@ -25,6 +28,11 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddDbContext<ApplicationContext>(opt=>
```
Comment (NerevitVasyan) [inline BobManager.API/BobManager.API/Startup.cs]: Format using spaces

File: BobManager.API/BobManager.API/Startup.cs [hunk 4]
```diff
@@ -25,6 +28,11 @@ public Startup(IConfiguration configuration)
...
+                opt.UseSqlServer(Configuration["ConnectionString"],
+                b=>b.MigrationsAssembly("BobManager.API"))
```
Comment (NerevitVasyan) [inline BobManager.API/BobManager.API/Startup.cs]: same

File: BobManager.API/BobManager.DataAccess/ApplicationContext.cs [hunk 5]
```diff
@@ -0,0 +1,42 @@
+﻿using BobManager.DataAccess.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace BobManager.DataAccess
+{
+    public class ApplicationContext : DbContext
+    {
+        public ApplicationContext(DbContextOptions<ApplicationContext> options) : base(options)
+        {
```
Comment (NerevitVasyan) [inline BobManager.API/BobManager.DataAccess/ApplicationContext.cs]: format like public ApplicationContext() { }

File: BobManager.API/BobManager.DataAccess/ApplicationContext.cs [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        public DbSet<SpendingCategory> SpendingCategories { get; set; }
+        public DbSet<ToDo> ToDos { get; set; }
+        public DbSet<ToDoCategory> ToDoCategories { get; set; }
+        public DbSet<User> Users { get; set; }
+        public DbSet<UsersGroup> UsersGroups { get; set; }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<UsersGroup>()
+                .HasKey(t => new { t.UserId, t.GroupId,t.GroupRoleId });
```
Comment (NerevitVasyan) [inline BobManager.API/BobManager.DataAccess/ApplicationContext.cs]: format

File: BobManager.API/BobManager.API/Startup.cs (hunk 3 above)
Comment (Liosh-a) [reply BobManager.API/BobManager.API/Startup.cs]: Resolved

File: BobManager.API/BobManager.DataAccess/ApplicationContext.cs (hunk 5 above)
Comment (Liosh-a) [reply BobManager.API/BobManager.DataAccess/ApplicationContext.cs]: Resolved

**Review by viviviviviivilll — APPROVED** [review]
Well done

**Review by NerevitVasyan — APPROVED** [review]
Good

**Review by ander21 — APPROVED** [review]
Approve

**Pull request merged**